components/UI/CopyToClipboard.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Clipboard from '../../assets/icons/clipboard.svg';
+import TextButton from './TextButton';
+
+const CopyToClipboard = ({ content, title }) => {
+    const [isContentCopied, setIsContentCopied] = useState(false);
```

Comment:
NAB: `wasContentCopied` is a more correct & declarative variable name, since there's no asynchronous action happening here. Instead the content was copied to the clipboard immediately after invoking the function.
We just set a timer ourselves... the `is` prefix IMHO makes sense when performing asynchronous actions like making a request to a server and waiting for the response.